core/common/src/main/java/alluxio/metrics/MetricKey.java

```diff
@@ -1340,6 +1340,40 @@ public MetricKey build() {
           .setMetricType(MetricType.GAUGE)
           .setIsClusterAggregated(false)
           .build();
+  public static final MetricKey WORKER_CACHE_MANAGER_THREAD_ACTIVE_COUNT =
+      new Builder("Worker.CacheMangerThreadActiveCount")
+          .setDescription("The approximate number of block cache "
+              + "threads that are actively executing tasks in the cache manger thread pool")
+          .setMetricType(MetricType.GAUGE)
+          .setIsClusterAggregated(false)
+          .build();
+  public static final MetricKey WORKER_CACHE_MANAGER_THREAD_CURRENT_COUNT =
+      new Builder("Worker.CacheMangerThreadCurrentCount")
+          .setDescription("The current number of cache threads in the cache manger thread pool")
+          .setMetricType(MetricType.GAUGE)
+          .setIsClusterAggregated(false)
+          .build();
+  public static final MetricKey WORKER_CACHE_MANAGER_THREAD_QUEUE_WAITING_TASK_COUNT =
+      new Builder("Worker.CacheMangerThreadQueueWaitingTaskCount")
+          .setDescription("The current number of tasks waiting in the work queue "
+              + "in the cache manager thread pool")
+          .setMetricType(MetricType.GAUGE)
+          .setIsClusterAggregated(false)
+          .build();
+  public static final MetricKey WORKER_CACHE_MANAGER_THREAD_MAX_COUNT =
```

Comment:
If this is actually bounded by a property key, i don't think we need to look at this?